libafl_targets/src/cmps/observers/aflpp.rs

```diff
@@ -106,7 +106,7 @@ where
         self.original
     }
 
-    /// Add [`struct@CmpValuesMetadata`] to the State including the logged values.
```

Comment:
Why not link this? like, use AFLppCmpValuesMetadata

Reply:
because i was lazy